contrib/datadog/tracing/interceptor.go

```diff
@@ -143,6 +143,18 @@ func (t *tracerImpl) ContextWithSpan(ctx context.Context, span interceptor.Trace
 	return tracer.ContextWithSpan(ctx, span.(*tracerSpan).Span)
 }
 
+// SpanFromWorkflowContext extracts the DataDog Span object from the workflow context.
+func SpanFromWorkflowContext(ctx workflow.Context) ddtrace.Span {
```

Comment:
We should match the equivalent [SpanFromContext](https://github.com/DataDog/dd-trace-go/blob/58e1e634e199676dd2ec5ce89600da91a282b253/ddtrace/tracer/context.go#L25) signature here, so return a ` (Span, bool)` and return a noop span if none is set

Reply:
@Quinn-With-Two-Ns how can I created a noop span? The `NoopSpan` is exported from `gopkg.in/DataDog/dd-trace-go.v1/ddtrace/internal` so it's not accessible outside the datadog tracing package

Reply:
We could implement our own NoopSpan here but this seems like a lot of extra code for little benefit

Reply:
I think `ddtrace.SpanFromContext(nil)` will do it

Reply:
Thank you for the suggestion, I have updated it to use this approach now

Reply:
Great thank you!